### ISSUE TITLE
Fanfiction.net covers fail to download

### ISSUE THREAD

**Issue by toroidal-code** [JimmXinu/FanFicFare#125]
Traceback:
```
FFF: INFO: 2016-07-17 16:49:03,668: story.py(1098): Failed to load or convert image, skipping:
https://ffcdn2012t-fictionpressllc.netdna-ssl.com/image/4028927/180/
Exception: HTTP Error 403: Forbidden
```

It's looking like FF.net wants the `Referer` field in the HTTP request to be something like `https://www.fanfiction.net/` + any valid path.

**Comment by JimmXinu** [comment]
I can't test that without knowing a story URL that refers to it...

**Comment by toroidal-code** [comment]
It's happening for every story I try. At this point I'm attempting random one-shots, but they're all acting the same.

```
~/Downloads ❯❯❯ fanficfare https://www.fanfiction.net/s/12037490/1/Asunder
You have include_images enabled, but Python Image Library(PIL) isn't found.
Images will be included full size in original format.
Continue? (y/n)?
y
FFF: INFO: 2016-07-17 17:30:02,490: story.py(1098): Failed to load or convert image, skipping:
https://ffcdn2012t-fictionpressllc.netdna-ssl.com/image/3877866/180/
Exception: HTTP Error 403: Forbidden
FFF: INFO: 2016-07-17 17:30:02,568: base_writer.py(179): Save directly to file: Asunder-ffnet_12037490.epub
```

**Comment by JimmXinu** [comment]
Interesting.  The ffnet adapter does set the Referer and has for a long time.  But the cookiejar caching code broke that, a somewhat lesser long time ago.  Thanks for pointing it out.

Fixed in: 6f4660763faf498877dc47eda3ab75c9c7c9970e
